.gitignore

```diff
@@ -0,0 +1,10 @@
+.ipynb_checkpoints
+__pycache__
+bin/combine_qc.py
+tests/combine_qc_test.py
```

Comment:
Just checking: this isn't a mistake that this test file is not under version control, correct?